next.config.mjs

```diff
@@ -2,8 +2,10 @@
 
 const nextConfig = {
   output: 'export',
-  distDir: 'build',
-  basePath: '/pets-frontend',
+
+  // comentar os dois abaixo para rodar localmemente, não esquecer de descomentar para subir o build
+  // distDir: 'build',
```

Comment:
não teria de descomentar agora que está subindo o build ?

Reply:
sim, mas só na master mano, aqui é QA

Reply:
Da pra fazer isso ai tb, mas ai seriam 2 ambientes, tem problema? a ideia que eu pensei seria nem ,mexer no arquivo mesmo saca, tipo na master fica descomentado sempre, e em QA fica comentado sempre e com isso ficando só com um ambiente no ar (MASTER) e um ambiente local mesmo (QA).
que ai só fazemos deplou mesmo quando for pra master